Utils/trigger_test_upload_flow.sh

```diff
@@ -63,59 +476,84 @@ while [[ "$#" -gt 0 ]]; do
   esac
 done
 
+# Setup
+cd "${CONTENT_PATH}" || fail
 
-if [ -z "$_ci_token" ]; then
-    echo "You must provide a ci token."
-    exit 1
+check_arguments
+
+# If production flag is set - upload master branch
+if [ -n "$production" ]; then
+  trigger_circle_ci "master"
+  exit 0
 fi
 
-if [ -n "$_force" ] && [ -z "$_packs" ]; then
-    echo "You must provide a csv list of packs to force upload."
-    exit 1
+
+new_content_branch="${sdk_branch_name}_${content_branch_name}_UploadFlow_test"
+new_suffix="New"
+new_pack_name="${base_pack_name}${new_suffix}"
+
+git checkout "$content_branch_name" || fail
+git pull || fail
+
+existed_in_remote=$(git ls-remote --heads origin "${new_content_branch}")
+existed_in_local=$(git branch --list "${new_content_branch}")
+
+# Deletes the remote branch if exists
+if [ -z "${existed_in_remote}" ]; then
+  git push origin --delete "${new_content_branch}"
+fi
+# Deletes the local branch if exists
+if [ -z "${existed_in_local}" ]; then
+  git branch -D "${new_content_branch}" # delete local branch
 fi
 
-if [ -n "$_gitlab" ]; then
+git checkout -b "${new_content_branch}" || fail "" "skip"
 
-  _variables="variables[BUCKET_UPLOAD]=true"
-  if [ -n "$_force" ]; then
-    _variables="variables[FORCE_BUCKET_UPLOAD]=true"
-  fi
+# Changes
+if [ -n "$sdk_branch_name" ]; then
+  change_sdk_requirements "${sdk_branch_name}" "dev-requirements-py3.txt"
+fi
 
-  source Utils/gitlab_triggers/trigger_build_url.sh
+# New Pack
+array=("Hello_World" "Hello World" "helloworld" "Sanity_Test")
+create_new_pack "${base_pack_name}" "${new_suffix}" "${array[@]}"
+add_dependency "${new_pack_name}" "Viper"
```

Comment:
```suggestion
add_dependency "Viper" "${new_pack_name}"
```
We want an old pack to be dependent in a new pack